app/views/admin/sponsors/edit.html.haml

```diff
@@ -1,12 +1,13 @@
-%section#admin#banner
+.containuer-fluid#admin#banner
```

Comment:
@KimberleyCook I think we could remove the `#admin` and `#banner` ids from here and instead apply spacing via a Bootstrap class. I've used `.pt-3` on other pages.